alpha/declcfg/load_test.go

```diff
@@ -890,17 +1073,27 @@ present in the .indexignore file.`),
 		"README.md":                readme,
 		"unrecognized-schema.json": unrecognizedSchema,
 	}
+
+	validFSD = fstest.MapFS{
+		".indexignore":     indexIgnore,
+		"cockroachdb.json": cockroachdb,
+		"etcd.yaml":        etcd,
+		"etcdoperator.v0.6.1.clusterserviceversion.yaml": etcdCSV,
+		"README.md":                readme,
+		"deprecations.yaml":        deprecations,
+		"unrecognized-schema.json": unrecognizedSchema,
+	}
 )
 
+type EvaluationFunc func(*testing.T, *DeclarativeConfig)
```

Comment:
Why don't the tests simply expose the `*DeclarativeConfig` they expect and use a `cmp.Diff()` on the output of the test?

Reply:
Attempting to leave a minimal footprint and just change the source of friction.  TestLoadFS already does validation over the generated `DeclarativeConfig`. In fact, TestLoadFile is probably superfluous, but I didn't think it was harmful to leave it.